From 54c323649e136690a9f978adfdca827b0eb843e7 Mon Sep 17 00:00:00 2001
From: Fikri Bachtiar <fikriachmadbachtiarr@gmail.com>
Date: Sat, 28 Sep 2024 21:38:23 +0700
Subject: [PATCH] rm: dont compile unused dts

---
 arch/arm/dts/Makefile | 30 +-----------------------------
 1 file changed, 1 insertion(+), 29 deletions(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 236cd2ac182..ce1ae53d227 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -28,36 +28,8 @@ dtb-$(CONFIG_EXYNOS5) += exynos5250-arndale.dtb \
 	exynos5422-odroidxu3.dtb
 dtb-$(CONFIG_EXYNOS7420) += exynos7420-espresso7420.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += \
-	rk3036-sdk.dtb \
-	rk3066a-mk808.dtb \
-	rk3126-evb.dtb \
-	rk3128-evb.dtb \
-	rk3188-radxarock.dtb \
-	rk3288-evb.dtb \
-	rk3288-fennec.dtb \
-	rk3288-firefly.dtb \
-	rk3288-miqi.dtb \
-	rk3288-phycore-rdk.dtb \
-	rk3288-popmetal.dtb \
-	rk3288-rock2-square.dtb \
-	rk3288-tinker.dtb \
-	rk3288-veyron-jerry.dtb \
-	rk3288-veyron-mickey.dtb \
-	rk3288-veyron-minnie.dtb \
-	rk3288-vyasa.dtb \
 	rk3308-evb.dtb \
-	rk3308-rock-pi-s.dtb \
-	rk3328-evb.dtb \
-	rk3368-lion.dtb \
-	rk3368-sheep.dtb \
-	rk3368-geekbox.dtb \
-	rk3368-px5-evb.dtb \
-	rk3399-evb.dtb \
-	rk3399-firefly.dtb \
-	rk3399-puma-ddr1333.dtb \
-	rk3399-puma-ddr1600.dtb \
-	rk3399-puma-ddr1866.dtb \
-	rv1108-evb.dtb
+	rk3308-rock-pi-s.dtb 
 dtb-$(CONFIG_ARCH_MESON) += \
 	meson-gxbb-odroidc2.dtb
 dtb-$(CONFIG_TEGRA) += tegra20-harmony.dtb \
